Create README.md for INGInious (#14)
blegat/LINMA2472: HomeworkAD/inginious/README.md

# Homework on INGInious

Two tasks are currently available on INGInious (https://inginious.info.ucl.ac.be/course/LINMA2472):

> - VectorizedReverse → corresponding to the first part of the project

Reviewer: ```suggestion
- [FirstOrder](https://inginious.info.ucl.ac.be/course/LINMA2472/FirstOrder) → corresponding to the first part of the project
```

> - SecondOrder → corresponding to the second part of the project

Reviewer: ```suggestion
- [SecondOrder](https://inginious.info.ucl.ac.be/course/LINMA2472/SecondOrder) → corresponding to the second part of the project
```

You are required to submit only your file reverse_vectorized.jl.

> A scoreboard is available on the platform. On the left-hand menu, you will find a “Scoreboard” tab where you can view the benchmark results for each task.

Reviewer: ```suggestion
A [scoreboard](https://inginious.info.ucl.ac.be/scoreboard/LINMA2472) is available on the platform. On the left-hand menu, you will find a “Scoreboard” tab where you can view the benchmark results for each task.
```
